app/(pages)/evidence/page.tsx

```diff
@@ -1,7 +1,22 @@
 import { FC } from 'react';
 
+import "./evidence.scss"
+import Form from '@/app/components/Evidence/form';
+import SideImage from '@/app/components/Evidence/sideImage';
+
+
+
 const Evidence: FC = () => {
-  return <div>Evidence</div>;
+  return  <div className="evidence__container"> 
```

Comment:
**evidence__container** is a very old pattern for naming classes please use hyphens(-) like **evidence-container**